ng-dev/release/notes/release-notes.ts

```diff
@@ -50,6 +60,28 @@ export class ReleaseNotes {
     return render(changelogTemplate, await this.generateRenderContext(), {rmWhitespace: true});
   }
 
+  /** Prepends the generated release note to the CHANGELOG file. */
```

Comment:
Nit: For good code quality I'd  personally recommend we also say that the changelog file is updated in the repository directory as provided by the Git client (part of the JSDoc description).

It's not clear where the file is updated if I just look at `ReleaseNotes#prependEntryToChangelog` from outside.